examples/remove_liquidity.rs

```diff
@@ -0,0 +1,94 @@
+   
+   let pair_address = "0xA6108E4d436bE592bAc12F9A0aB7D9A10d821176".parse::<Address>()?;
+   let swap_address = "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D".parse::<Address>()?;
+
+//    let liquidity = U256::from_dec_str("50000000")?;
+//    println!("data {}", liquidity);
+   
+   // connect to the network
+   let provider = Provider::<Http>::try_from("https://rinkeby.infura.io/v3/a111fcada47746d990e0e2b7df50d00a").unwrap();
+   
+   // create the contract object at the address
+   let pair_contract = Contract::new(pair_address, pair_abi, provider.clone());
+   let swap_contract = Contract::new(swap_address, swap_abi, provider.clone());
+   
+   let wallet = wallet.with_chain_id(4u64);
+   let client = SignerMiddleware::new(provider, wallet);
+   let client = Arc::new(client);
+
+   let (reserve0, reserve1, _) = pair_contract
+    .method::<_, (u128, u128, u32)>("getReserves", ())?
+    .call()
+    .await?;
+    
+    println!("Reserves (token A, Token B): ({}, {})", reserve0, reserve1);
+
+    let mut price = 0;
+
+    if reserve0 > reserve1 {
+        price = reserve0/reserve1;
+    } else{
+        price = reserve1/reserve0;
+    }
+
+    println!("token price in % = {}", price);
+
+    // if (price > 10){
+        println!("Apporoving the transaction!");
+        
+        let contract_with_wallet = pair_contract.connect(Arc::clone(&client));
+        
+        let liquidity = U256::from_dec_str("100")?;
+
+        let call = contract_with_wallet
+            .method::<_, H256>("approve", (swap_address, liquidity.clone()))?;
+
+        let pending_tx = call.send().await?;
+        let receipt = pending_tx.confirmations(6).await?;
```

Comment:
6 confirmations seem excessive in an example

Reply:
done